src/sql/src/kafka_util.rs

```diff
@@ -497,15 +497,24 @@ pub fn generate_ccsr_client_config(
 ) -> Result<ccsr::ClientConfig, anyhow::Error> {
     let mut client_config = ccsr::ClientConfig::new(csr_url);
 
-    if let Some(ca_path) = kafka_options.get("ssl.ca.location") {
+    if let Some(ca_path) = match ccsr_options.get("ssl_ca_location") {
+        Some(Value::String(path)) => Some(path),
+        _ => kafka_options.get("ssl.ca.location"),
+    } {
```

Comment:
`Option::or_else` simplifies this code (and the similar code below)

I would also add a comment here explaining what is going on
```suggestion
    if let Some(ca_path) = ccsr_options.get("ssl_ca_location").or_else(|| {
        kafka_options.get("ssl.ca.location"),
    }) {
```

Reply:
`ccsr_options` and `kafka_options` map to different value types (`Value` and `String` respectively), so I'm using a `match`. I've added `bail!` calls as we've discussed over Slack.